### PR TITLE
Created CONTRIBUTING.md doc with 'Maintainers' section

### PR DESCRIPTION
**Pull request by calebkleveter**

### CODE REVIEW

**Review by MrLotU — CHANGES_REQUESTED** [review]
Could you copy nio-kit's CONTRIBUTING.md's contents in this one too? So that it provides actual Contributing guidelines too?

**Comment by calebkleveter** [comment]
@MrLotU Added!

**Review by MrLotU — APPROVED** [review]

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Contributing
+
+We're excited to see you're interested in contributing to the CodableKit package! This document covers how you can report any issues you find or contribute with bug fixes and new features!
+
+## Reporting Issues
+
+Go ahead and [open a new issue](https://github.com/vapor-community/nio-kit/issues/new). The team will be notified and we should get back to you shortly.
```
Comment (tanner0101) [inline .github/CONTRIBUTING.md]: wrong link for the new issue here.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Contributing
```
Comment (tanner0101) [inline .github/CONTRIBUTING.md:1]: for markdown docs, we should try to keep line length at a max of 70. helps a lot with diffs!

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## Pull Requests
+
+We are always willing to have pull requests open for bug fixes or new features.
+
+Bug fixes are pretty straight forward. Open up a PR and a maintainer will help guide you to make sure everything is in good order.
+
+New features are little more complex. If you come up with an idea, you should follow these steps:
+
+- [Open a new issue](https://github.com/vapor-community/nio-kit/issues/new) or post in the #ideas or #development channel on the [Discord server](http://vapor.team/). This lets the us know what you have in mind so we can discuss it from a higher level down to the implementation details.
```
Comment (tanner0101) [inline .github/CONTRIBUTING.md]: link wrong

**Review by tanner0101 — CHANGES_REQUESTED** [review]
Just some small issues, but overall looks great 👍

**Comment by calebkleveter** [comment]
@tanner0101 👍

**Review by tanner0101 — APPROVED** [review]

**Review by tanner0101 — APPROVED** [review]

**Comment by penny-coin** [comment]
Hey @calebkleveter, you just merged a pull request, have a coin! 

You now have 437 coins.

**Pull request merged**